azurelinuxagent/common/osutil/default.py

```diff
@@ -1354,7 +1354,7 @@ def get_total_cpu_ticks_since_boot():
         if proc_stat is not None:
             for line in proc_stat.splitlines():
                 if ALL_CPUS_REGEX.match(line):
-                    system_cpu = sum(int(i) for i in line.split()[1:7])
+                    system_cpu = sum(int(i) for i in line.split()[1:8])
```

Comment:
Can you add a comment on what the columns are?

Reply:
OK - I'll add a comment referencing what man page can be consulted (description can be sort of long to include here)